test/pokemon-legality-checker.spec.js

```diff
@@ -24,8 +24,22 @@ describe('pokemon-legality-checker', () => {
         const parsedFile = pk6parse.parseFile(`${__dirname}/pk6/illegal/${filename}`, {parseNames: true});
         const check = legalityCheck(parsedFile);
         expect(check.isLegal).to.be.false(`Incorrectly classifies the illegal Pokémon ${filename} as legal`);
-        expect(check.reason).to.be.a('string');
+        expect(check.errors.length).to.not.be.equal(0);
+        expect(check.errors[0].field).to.be.a('string');
```

Comment:
Oh, and I've found one.

'contest' isn't a field on the pokemon. But I'm not sure we'd want to use any of the contest fields on it.